pkg/user/tx_client.go

```diff
@@ -395,62 +409,13 @@ func (client *TxClient) broadcastTx(ctx context.Context, txBytes []byte, signer
 	return resp.TxResponse, nil
 }
 
-// retryBroadcastingTx creates a new transaction by copying over an existing transaction but creates a new signature with the
-// new sequence number. It then calls `broadcastTx` and attempts to submit the transaction
-func (client *TxClient) retryBroadcastingTx(ctx context.Context, txBytes []byte) (*sdktypes.TxResponse, error) {
-	blobTx, isBlobTx, err := blobtx.UnmarshalBlobTx(txBytes)
-	if isBlobTx {
-		// only check the error if the bytes are supposed to be of type blob tx
-		if err != nil {
-			return nil, err
+// pruneTxTracker removes transactions from the local tx pool that are older than 10 minutes
```

Comment:
nit: elsewhere in this PR it's called a txTracker. Can we use that term consistently instead of tx pool sometimes and txTracker other times.

```suggestion
// pruneTxTracker removes transactions from the local tx tracker that are older than 10 minutes
```